packages/invoices/src/InboundShipment/DetailView/reducer/reducer.ts

```diff
@@ -112,15 +112,15 @@ export const createSummaryItem = (
     itemId: itemId,
     itemName: ifTheSameElseDefault(batches, 'itemName', ''),
     itemCode: ifTheSameElseDefault(batches, 'itemCode', ''),
-    itemUnit: ifTheSameElseDefault(batches, 'itemUnit', ''),
+    // itemUnit: ifTheSameElseDefault(batches, 'itemUnit', ''),
     batches: arrayToRecord(batches),
     unitQuantity: batches.reduce(getUnitQuantity, 0),
     numberOfPacks: batches.reduce(getSumOfKeyReducer('numberOfPacks'), 0),
-    locationDescription: ifTheSameElseDefault(
-      batches,
-      'locationDescription',
-      undefined
-    ),
+    // locationDescription: ifTheSameElseDefault(
+    //   batches,
+    //   'locationDescription',
+    //   undefined
+    // ),
```

Comment:
once I made the keys of the domain objects actually start getting checked, a decent amount of compiler errors came up, using fields which weren't actually on the types.. 🙄

Reply:
it works! thanks typescript 😃